Makefile

```diff
@@ -356,6 +356,16 @@ sr-snapshot-tests-open:
 	@$(ECHO_TITLE) "make sr-snapshot-tests-open"
 	./tools/sr-snapshot-test.sh --open-project
 
+# Define default paths for API output files
+SWIFT_OUTPUT_PATH ?= api-surface-swift
+OBJC_OUTPUT_PATH ?= api-surface-objc
+
+# Use different paths when running in CI
+ifeq ($(ENV),ci)
+  SWIFT_OUTPUT_PATH := api-surface-swift-generated
+  OBJC_OUTPUT_PATH := api-surface-objc-generated
+endif
+
 # Generate api-surface files for Datadog and DatadogObjc.
 api-surface:
 		@echo "Generating api-surface-swift"
```

Comment:
suggestion/ Let's follow the convention in of [printing command's title before it is ran](https://github.com/DataDog/dd-sdk-ios/blob/004ca93c6036c0ceead2f09fedebfae9496c9da3/Makefile#L313). This brings a lot of clarity to CI logs:
```
	@$(ECHO_TITLE) "make api-surface"
```

Also, let's fix the indentation for this command. It uses the old one, all new commands (after migration to GitLab) are indented with less whitespaces. See [these examples](https://github.com/DataDog/dd-sdk-ios/blob/004ca93c6036c0ceead2f09fedebfae9496c9da3/Makefile#L311-L328).